gravity-dex.md

```diff
@@ -173,17 +174,21 @@ gaiad version
     # global HTTP write timeout, which applies to all connections and endpoints.
     # See https://github.com/tendermint/tendermint/issues/3435
     timeout_broadcast_tx_commit = "600s"
+    ```
 
+2. Another change to make in the `$HOME/.gaia/config/config.toml` file is to increase the timeout for a transaction to be committed and maximum size of request body and header. Since IBC and swap state transactions are very large, your node might stop from time to time when your node receives too many requests. Configure these values to increase the timeout: 
+
+    ```bash
     # Maximum size of request body, in bytes
     max_body_bytes = 10000000
 
     # Maximum size of request header, in bytes
     max_header_bytes = 10485760
     ```
 
-    Note: If your node is unable to connect to any of the seeds listed here, you can find seeds and peers in [this document](https://hackmd.io/@KFEZk8oMTz6vBlwADz0M4A/BkKEUOsZu#) maintained by community members, and at [Atlas](https://atlas.cosmos.network/nodes), which is automatically generated by crawling the network. You can also join [Discord channel](https://discord.gg/vcExX9T) to reach out to any Cosmos community members to get help on this.
+    Note: If your node is unable to connect to any of the seeds listed here, you can find seeds and peers in the [stargate peers and seed nodes](https://hackmd.io/@KFEZk8oMTz6vBlwADz0M4A/BkKEUOsZu#) document that is maintained by community members and in the [Atlas node explorer](https://atlas.cosmos.network/nodes), a list that is automatically generated by crawling the network. You can also join the [Discord channel](https://discord.gg/vcExX9T) to reach out to Cosmos community members to get help.
 
-2. Replace necessary configuration 
+3. Run these commands to configure the seed nodes:
```

Comment:
instead of "Replace necessary configuration" I think we're saying to run these commands?

Reply:
Yes. I think it is better to say "Run these commands to configure the seed nodes". I'll make that change.